include/spirv/unified1/spirv.core.grammar.json

```diff
@@ -11596,6 +11596,33 @@
           ],
           "capabilities" : [ "VectorComputeINTEL" ],
           "version" : "None"
+        },
+        {
+          "enumerant" : "MaximumRegistersINTEL",
+          "value" : 6461,
+          "parameters" : [
+            { "kind" : "LiteralInteger", "name" : "'Number of Registers'" }
+          ],
+          "capabilities" : [ "RegisterLimitsINTEL" ],
+          "version" : "None"
+        },
+        {
+          "enumerant" : "MaximumRegistersIdINTEL",
+          "value" : 6462,
+          "parameters" : [
+            { "kind" : "IdRef", "name" : "'Number of Registers'" }
+          ],
+          "capabilities" : [ "RegisterLimitsINTEL" ],
+          "version" : "None"
+        },
+        {
+          "enumerant" : "NamedMaximumRegistersINTEL",
+          "value" : 6463,
+          "parameters" : [
+            { "kind" : "LiteralInteger", "name" : "'Named Maximum Number of Registers'" }
```

Comment:
Question goes to @bashbaug , with the spelling from https://github.com/KhronosGroup/SPIRV-Registry/pull/235 `"kind" : "LiteralInteger"` seems to be correct. But I have a feeling, that intention here should be to change kind to `"kind" : "NamedMaximumNumberofRegisters"` binding the parameter with possible values to be used by validator.

Reply:
Good catch - I do think this would make validation easier, and the disassembled SPIR-V might be a little nicer also.

Can you have a look at my suggested change on the spec PR to be sure this is what you're thinking of?
https://github.com/KhronosGroup/SPIRV-Registry/pull/235#discussion_r1484906698

Let me know if you need help with any other tooling, also.

Reply:
I accepted the spec suggestion.  Let's go with the "NamedMaximumNumberOfRegisters" "kind".  Thanks!

Reply:
Create a PR to Tools to support operand kind there too: https://github.com/KhronosGroup/SPIRV-Tools/pull/5580